changelogs/fragments/2174-ipa-user-userauthtype-multiselect.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This is more a new feature than a bugfix.
```suggestion
minor_changes:
```